control-plane-operator/hostedclusterconfigoperator/controllers/resources/kas/admissionpolicies.go

```diff
@@ -79,6 +88,9 @@ func reconcileConfigValidatingAdmissionPolicy(ctx context.Context, hcp *hyperv1.
 	if hcp.Spec.OLMCatalogPlacement == hyperv1.ManagementOLMCatalogPlacement {
 		configResources = append(configResources, "operatorhubs")
 	}
+	if !isOnPremPlatform(hcp.Spec.Platform.Type) {
```

Comment:
could we instead allow the cno client https://github.com/openshift/hypershift/pull/4303/files#diff-b6b6ca31afcd6a987e7138f0d8d83874e551721abca008aad75f0d30cbb3eed2R36-R37?

Reply:
> which could be further restricted to the service account of the CNO. However, we just want to get Openstack back online asap for now.

Yeah, we could.  Just wanted to get it working again vs complicating the code too much.